user/src/main/java/bisq/user/identity/UserIdentityService.java

```diff
@@ -214,7 +214,7 @@ public boolean hasUserProfiles() {
     }
 
     public Observable<UserIdentity> getSelectedUserIdentity() {
-        return persistableStore.getSelectedUserIdentity();
+        return persistableStore.getSelectedUserIdentityObservable();
```

Comment:
If we follow the JavaFx Property pattern it would be `selectedUserIdentityObservable()` . I was never a big fan of that convention but I guess it might be still the better approach to keep it similar to Properties and apply also the JavaFx convention where we use properties (a lot). I don't think its an urgent refactoring task, but can be done over time. Maybe ChatGPT can do it ;-) its really a code monkey job ;-).